Formula/lensfun@0.3.95.rb

```diff
@@ -0,0 +1,43 @@
+class LensfunAT0395 < Formula
+  include Language::Python::Shebang
+
+  desc "Remove defects from digital images"
+  homepage "https://lensfun.github.io/"
+  url "https://downloads.sourceforge.net/project/lensfun/0.3.95/lensfun-0.3.95.tar.gz"
+  sha256 "82c29c833c1604c48ca3ab8a35e86b7189b8effac1b1476095c0529afb702808"
+  license all_of: [
+    "LGPL-3.0-only",
+    "GPL-3.0-only",
+    "CC-BY-3.0",
+    :public_domain,
+  ]
+  revision 4
```

Comment:
New formula should not have a revision.

Reply:
So you are saying as we are creating the first revision of the new formula "lensfun@0.3.95.rb" it should not have a revision, although it is a literal copy of revision 4 of the default lensfun formula. Will remove the revision.